src/content/docs/apm/apm-ui-pages/events/record-deployments.mdx

```diff
@@ -321,23 +321,21 @@ Some agents have additional methods to record deployments:
 * **Python**: Use the [`record-deploy`](/docs/agents/python-agent/installation-configuration/python-agent-admin-script#record-deploy) subcommand of the `newrelic-admin` script.
 * **Ruby**: Use a [Capistrano recipe](/docs/agents/ruby-agent/features/recording-deployments-ruby-agent#capistrano3).
 
-## Notify your team of deployments [#webhooks]
+## Notify your team about deployments [#webhooks]
 
-After a deployment is recorded using the REST API, you can optionally notify a webhook endpoint of the deployment.
+After a deployment is recorded, through either the REST API or the newer [GraphQL API](/docs/change-tracking/change-tracking-introduction), you can notify your team members using a webhook.
 
-The destination of the webhook can be your Slack instance. To use webhooks to set up a deployment notification for a Slack channel:
+While you can send deployment data to any webhook destination, here are the steps to set this up with Slack:
```

Comment:
@jonathanpdx I changed this sentence to this: "While you can send deployment data to any webhook destination, if you want to use a Slack webhook, make sure you use the legacy New Relic Alerts app:"

Reply:
My concern is that, "While you can send deployment data to any webhook destination" is not entirely true. If you attempt to send to a modern Slack webhook destination, it will fail. I can see customers saying, "I already have a perfectly fine Slack webhook, I'd rather not go through the bother of integrating with their legacy Slack app." And then, sadness.

Reply:
@jonathanpdx I was thinking that readers would see this sentences as meaning "any webhook destination with the exception of the special condition for Slack." Does this make it more explicit: "You can send deployment data to a variety of webhook destinations. Simply follow the instructions for your whatever tool you're using—except Slack. If you want to use a Slack webhook, you must use the legacy New Relic Alerts app instead of the New Relic Slack app like this:"?